File: javascript/lanchonete.js
const codigoProduto = parseInt(prompt("Digite o código do produto"));
const qtdComprada = parseInt(prompt("Digite a quantidade comprada"));

switch (codigoProduto) {
    case 1:
        valorTotal = 5 * qtdComprada;
        break;
    case 2:
        valorTotal = 3.5 * qtdComprada;
        break;
    case 3:
        valorTotal = 4.8 * qtdComprada;
        break;
    case 4:
        valorTotal = 8.9 * qtdComprada;
        break;
    case 5:
        valorTotal = 7.32 * qtdComprada;
        break;
    default:
        valorTotal = -1;
        break;
}

console.log("Valor total = R$ %.2f", valorTotal);

File: python/lanchonete.py
codigo = int(input("Codigo do produto comprado: "))
quantidade = int(input("Quantidade comprada: "))
valor_unitario = 0

if codigo == 1:
    valor_unitario = 5
elif codigo == 2:
    valor_unitario = 3.5
elif codigo == 3:
    valor_unitario = 4.8
elif codigo == 4:
    valor_unitario = 8.9
elif codigo == 5:
    valor_unitario = 7.32

valor_total = valor_unitario * quantidade

print(f"Valor a pagar: R$ {valor_total:.2f}")
